DataManagementSystem/scripts/dirac-dms-replicate-and-register-request.py

```diff
@@ -12,7 +14,8 @@
                                      '    targetSE: target SE' ] ) )
 
 catalog = None
-Script.registerSwitch( "C:", "Catalog=", "   Catalog to use" )
+Script.registerSwitch("C:", "Catalog=", "Catalog to use")
```

Comment:
@chaen: I changed the order of calls to parseCommandLine and getUnprocessedSwitches, because otherwise the output lines are prepended with the date and loglevel, etc, which doesn't look nice